stdlib/experimental/oee/oee.flux

```diff
@@ -0,0 +1,46 @@
+package oee
+
+import "contrib/tomhollingworth/events"
+import "experimental"
+
+computeAPQ = (
+    productionEvents,
+    partEvents,
+    runningState,
+    plannedTime,
+    idealCycleTime
+) => {
+    availability = productionEvents
+        |> events.duration(unit: 1ns, columnName: "runTime")
+        |> filter(fn: (r) => r.state == runningState)
+        |> sum(column: "runTime")
+        |> duplicate(column: "_stop", as: "_time")
+        |> map(fn: (r) => ({ r with availability: float(v: r.runTime) / float(v: int(v: plannedTime)) }))
```

Comment:
small nit, you can do the `duplicate` step as part of the `map` step to make it a bit faster. 
```
|> map(fn: (r) => ({ r with _time: r._stop, availability: float(v: r.runTime) / float(v: int(v: plannedTime)) }))
```